app/controllers/api/v1/visualization/clusters_controller.rb

```diff
@@ -235,8 +235,10 @@ def self.get_cluster_viz_data(study, cluster, url_params)
           }
 
           coordinate_labels = ClusterVizService.load_cluster_group_coordinate_labels(cluster)
-          custom_colors = cluster.study_file.cluster_file_info&.decoded_custom_colors || {}
+          custom_colors = cluster.study_file.cluster_file_info&.custom_colors_as_hash || {}
           custom_annotation_colors = custom_colors[annotation[:name]] || {}
+          annotation_split_defaults = cluster.study_file.cluster_file_info&.annotation_split_defaults_as_hash || {}
+          split_label_arrays = annotation_split_defaults[annotation[:name]] || false
```

Comment:
Since `nil` and `false` both evaluate as "falsey":
```suggestion
          split_label_arrays = annotation_split_defaults[annotation[:name]].present?
```

Reply:
Oh that is interesting -- I did not realize `false.present?` was `false`!  The code shortening is a little too much dark magic for me, but I'm very glad I know that fact now